app/assets/stylesheets/vendor/_select2.sass

```diff
@@ -0,0 +1,422 @@
+.select2-container
```

Comment:
Should we move the new _select2.sass to the vendor/assets or is it better to move there the original select2.css file?